.pre-commit-config.yaml

```diff
@@ -163,12 +160,14 @@ repos:
     rev: v0.38.0
     hooks:
       - id: markdownlint
-        args: [ '--disable',
-                MD013,  # Line length can be ignored for now
-                MD033,  # Allow inline HTML
-                MD046,  # Allow code blocks to be fenced with backticks
-                MD041,  # Allow multiple top level headers
-                '--' ]
+        args: [
+            "--disable",
+            MD013, # Line length can be ignored for now
+            MD033, # Allow inline HTML
+            MD046, # Allow code blocks to be fenced with backticks
+            MD041, # Allow multiple top level headers
+            "--",
+          ]
```

Comment:
@orouz I don't see any markdown changes in this PR, did you have any issue with this linter?

Reply:
auto-format issue. fixed.